osu.Game/Overlays/BeatmapListing/SearchRank.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.ComponentModel;
+
+namespace osu.Game.Overlays.BeatmapListing
+{
+    public enum SearchRank
```

Comment:
Skeptical this needs to exist alongside `ScoreRank`. Would just use that instead and provide a way to override `ScoreRank`'s `Description`s.